wa/workloads/speedometer/__init__.py

```diff
@@ -131,6 +131,10 @@ def initialize(self, context):
                 "Device must be rooted for the speedometer workload currently"
             )
 
+        if not self.target.package_is_installed(self.chrome_package):
+            raise WorkloadError(
+                "Could not find '{}' on the device. Please ensure it is installed.")
```

Comment:
I would add `", or specify the correct package name using 'chrome_package' parameter."`